testrunner-core-plugin/src/main/scala/edu/illinois/cs/testrunner/coreplugin/TestPluginUtil.java

```diff
@@ -0,0 +1,94 @@
+package edu.illinois.cs.testrunner.coreplugin;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.util.logging.Logger;
+import java.util.Arrays;
+import java.util.List;
+import java.util.logging.Level;
+
+import edu.illinois.cs.testrunner.util.ProjectWrapper;
+import edu.illinois.cs.testrunner.configuration.ConfigProps;
+import edu.illinois.cs.testrunner.configuration.Configuration;
+
+public class TestPluginUtil {
+    final public static String pluginName = "testplugin";
+    final public static String pluginClassName = "testplugin.className";
```

Comment:
Are these hardcoded final public static strings needed?

Reply:
Hardcoded final public static strings are needed as they will be shared/used in both gradle plugin and maven plugin to identify the pluginName and also the pluginClassName (e.g. idflakies' DetectorPlugin or ifixflakies' CleanerFixerPlugin)